### DISCUSSION TITLE
I can't find a option to sort students that can geneate barrier.Maybe I just can't find it?

### DISCUSSION THREAD

**Discussion by ShiinaRio** [lonqie/SchaleDB#84] [Ideas and Suggestions]
Was playing Goz raid and find out.And maybe a option for sorts students that can change position or geneate cover?

**Comment by lonqie** [comment]
A "barrier" option exists under the Buffs filter. Filter for reposition or cover summoners do not exist
![image](https://github.com/user-attachments/assets/69f0532e-46f3-4500-a540-073c28d3eb44)
![image](https://github.com/user-attachments/assets/fb69cb7f-c717-4701-ac69-688c9c44220a)

**Comment by ShiinaRio** [comment]
I see,this is only from the new site.I appreciate your effort.Keep up the good work!
